benchmark: add --runs support to run.js (#55158)
nodejs/node: doc/contributing/writing-and-running-benchmarks.md

node benchmark/run.js assert async_hooks
```

It's also possible to execute the benchmark more than once using the
`--runs` flag.

```bash
node benchmark/run.js --runs 10 assert async_hooks
```

> This command will run the benchmark suite 10 times for both assert and
> async\_hooks group

Reviewer: ```suggestion
This command will run the benchmark files in `benchmark/assert` and `benchmark/async_hooks`
10 times each.
```
